cmd/kubectl-directpv/kubectl-direct_csi.go

```diff
@@ -43,7 +43,7 @@ func main() {
 		os.Exit(1)
 	}()
 
-	if filepath.Base(os.Args[0]) == "kubectl-direct_csi" {
+	if filepath.Base(os.Args[0]) == "kubectl-directpv" {
```

Comment:
this checks for old binary name and shows the deprecation notice. The check should be for direct_csi, not directpv.